packages/app-next/src/App.tsx

```diff
@@ -83,6 +94,24 @@ const homePageExtension = createExtension({
   },
 });
 
+const scmAuthExtension = createApiExtension({
+  factory: ScmAuth.createDefaultApiFactory(),
+});
+
+const scmIntegrationApi = createApiExtension({
+  factory: createApiFactory({
+    api: scmIntegrationsApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => ScmIntegrationsApi.fromConfig(configApi),
+  }),
+});
+
+const collectedLegacyPlugin = collectLegacyRoutes(
```

Comment:
minimal suggestion as it could a collection of multiple routes of plugins

```suggestion
const collectedLegacyPlugins = collectLegacyRoutes(
```